airflow/ui/src/app.test.tsx

```diff
@@ -17,14 +17,15 @@
  * under the License.
  */
 
-import { afterEach, beforeEach, describe, it, vi } from "vitest";
-import { App, AppSimple } from "./app";
 import { render } from "@testing-library/react";
+import { afterEach, beforeEach, describe, it, vi } from "vitest";
+import { App } from "./app";
+import { AppSimple } from "./AppSimple";
 
+import type { QueryObserverSuccessResult } from "@tanstack/react-query";
 import * as openapiQueriesModule from "openapi/queries";
-import { Wrapper } from "./utils/test";
-import { DAGCollection } from "openapi/requests/types.gen";
-import { QueryObserverSuccessResult } from "@tanstack/react-query";
+import type { DAGCollection } from "openapi/requests/types.gen";
+import { Wrapper } from "./utils/Wrapper";
```

Comment:
Imports are organized automatically now.

Reply:
What's the ordering rule? I see local imports above package imports.

Reply:
Is just alphabetical. I can add the prettier plugin to sort them by groups, atm is just using the organize imports feature of ESLint itself ☺️

Reply:
I think grouping would be great! npm packages, relative paths and absolute paths

Reply:
Amazing! I'll do that first thing tomorrow! ✨

Reply:
Fixed [here](https://github.com/apache/airflow/pull/42184/commits/cf29ca2a0cec9cdb17bf42f32937de392686763c)